src/App.js

```diff
@@ -72,6 +73,34 @@ function App() {
     setNotes(notes)
     onCloseModal()
   }
+  const [openDeleteModal, setOpenDeleteModal] = useState({
```

Comment:
openDeleteModal indicates that this is a boolean. If you want to store more values, please find a better name.